src/audio/module_adapter/module/iadk_modules.c

```diff
@@ -44,6 +46,7 @@ DECLARE_SOF_RT_UUID("iadk_modules", intel_uuid, 0xee2585f2, 0xe7d8, 0x43dc,
 		    0x90, 0xab, 0x42, 0x24, 0xe0, 0x0c, 0x3e, 0x84);
 DECLARE_TR_CTX(intel_codec_tr, SOF_UUID(intel_uuid), LOG_LEVEL_INFO);
 
+static uint8_t is_loadable;
```

Comment:
Whats the use for this ? it seems like we are missing a handle structure for this API that would allow us to carry context across API calls ?

Reply:
Correct. I had yesterday discussion with @serhiy-katsyuba-intel about it and how to handle it. His suggestion is to add position in module interface with information what kind of library we are dealing with.

Reply:
I've changed method discovering what module is being loaded by adding bool variable to processing module. I think that it is cleaner now.